src/Pdf/CakePdf.php

```diff
@@ -738,6 +752,40 @@ public function template($template = false, $layout = null)
     }
 
     /**
+     * Template path
+     *
+     * @param mixed $templatePath The path of the template to use
+     * @return mixed
+     */
+    public function templatePath($templatePath = false)
```

Comment:
should I change the others too then? (https://github.com/FriendsOfCake/CakePdf/blob/master/src/Pdf/CakePdf.php#L726)